frontend/src/components/home/carousel.jsx

```diff
@@ -1,12 +1,30 @@
 import React, { Component } from "react";
 import { Carousel, Navbar } from "react-bootstrap";
+import ymim1 from "../../assets/ymim1.png";
+import ymim5 from "../../assets/ymim5.png";
+import ymim6 from "../../assets/ymim6.png";
+import ymim7 from "../../assets/ymim7.png";
 import "./index.css";
 import ymim from "../../assets/ymim6.jpeg";
 import logo from "../../assets/logo.png";
+import "./index.css";
 
 class MainCarousel extends Component {
-  render() {
+
+  static CarouselItem(image, altText) {
```

Comment:
i see you are following Matt's comment but this doesn't compile. 

I think what we want is to create a CarousalItem Component that takes in the image and title as props 

Let me know if you want to pair on this